pom.xml

```diff
@@ -40,10 +40,29 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
For anyone else who doesn't remember - we're bumping the minimum Java version from 6 to 7, because the runtime was actually already incompatible with Java 6. See https://github.com/kaitai-io/kaitai_struct_java_runtime/issues/34#issuecomment-745699504